core/src/main/scala/coulomb/conversion/standard/implicitConversion.scala

```diff
@@ -27,7 +27,29 @@ object implicitConversion:
     // I gate this with AllowImplicitConversions because many operations on coulomb
```

Comment:
is `AllowImplicitConversions` part of coulomb or scalalib?

Reply:
`AllowImplicitConversions` it is a coulomb thing I defined.

Scala has `scala.language.implicitConversions` which you can import, but it isn't an implicit var so I can't use it.